### DISCUSSION TITLE
Can I use signInWithIdToken with azure?

### DISCUSSION THREAD

**Discussion by AaronRivera-workdev** [supabase/supabase#18546] [Questions]
I am attempting to authenticate a user from a single page app. I have a an Entra app in azure that is linked to my supabase, oauth works as expected.

My issue arises when I attempt to authenticate with  "signInWithIdToken".  I've retrieved an ID token using MSAL on the client, but when I call "signInWithIdToken" I see the following error on our server: "Unhandled server error: Get \"/.well-known/openid-configuration\". 

Attempting to follow the code, it seems that issuer can't possibly be set correctly here: https://github.com/supabase/gotrue/blob/23852122f842dcb34ae47cc95fe28467d031bc70/internal/api/token_oidc.go#L57C14-L57C20

Am I thinking about this correctly? In what scenarios does "signInWithIdToken" work with azure?

**Comment by GaryAustin1** [comment]
Looking a level deeper it seems azure is handled differently so I don't think the code you pointed to is the issue.
https://github.com/supabase/gotrue/blob/23852122f842dcb34ae47cc95fe28467d031bc70/internal/api/provider/oidc.go#L56
It is run as a generic issuer and processed differently.
Got nothing else helpful though.

You might look thru some of the issues.  https://github.com/supabase/gotrue/issues?q=is%3Aissue+azure+is%3Aclosed
